HerPublicWebsite.BusinessLogic/Services/QuestionFlow/QuestionFlowService.cs

```diff
@@ -357,7 +357,7 @@ private QuestionFlowStep NotParticipatingForwardDestination(Questionnaire questi
         
         private QuestionFlowStep PendingForwardDestination(Questionnaire questionnaire)
         {
-            return QuestionFlowStep.Pending;
+            return QuestionFlowStep.HouseholdIncome;
```

Comment:
⚠️ I don't think that this flow is always correct.

`SelectAddressForwardDestination()` (which is the point we should be picking up from) has the following options for next steps:
```
else if (questionnaire.FoundEpcBandIsTooHigh)
{
    return QuestionFlowStep.ReviewEpc;
}
// If the LA has changed and the income band the user selected previously is no longer valid then we don't
// go back to the check your answers page as the user will need to select a new income band.
else if (entryPoint is QuestionFlowStep.Address && questionnaire.IncomeBandIsValid)
{
    return QuestionFlowStep.CheckAnswers;
}

return QuestionFlowStep.HouseholdIncome;
```

I think we need to clarify with DESNZ what we should be doing if we should be showing the `ReviewEpc` step.

I would expect that we should show that in preference to the Pending page.

Reply:
For now, I've moved the ReviewEPC check so it catches people before they're caught by the Pending check :)